### PR TITLE
add @andrewsykim, @neoli123, @stevesloka and @johnSchnake as test/ reviewers

### PR DESCRIPTION
**Pull request by andrewsykim**
Signed-off-by: Andrew Sy Kim <kiman@vmware.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
As part of https://github.com/kubernetes/kubernetes/issues/75601, we would like to help review more tests. 

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/assign @timothysc

### CODE REVIEW

**Comment by andrewsykim** [comment]
cc @stevesloka @neolit123

File: test/OWNERS [hunk 1]
```diff
@@ -32,6 +32,9 @@ reviewers:
   - MaciekPytel # for test/e2e/common/autoscaling_utils.go
   - oomichi
   - xichengliudui
+  - andrewsykim
+  - neolit123
+  - stevesloka
```
Comment (timothysc) [inline test/OWNERS]: Could you add john Schnake too as a reviewer here?

**Review by timothysc — APPROVED** [review]
/approve 

update then lgtm from me.

**Comment by timothysc** [comment]
@johnSchnake

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**